constants.cs

```diff
@@ -89,7 +89,8 @@ public enum TYPES {
 public enum KINDS {
     VAR       =     TOKENS.VAR,
     PROCEDURE =     TOKENS.PROCEDURE,
-    FUNCTION  =     TOKENS.FUNCTION
+    FUNCTION  =     TOKENS.FUNCTION,
+    PARAMETER
```

Comment:
This is just for the symbol table stuff.. I'm not saying that will cause any issues yet. But we don't know exactly how we're going to use this yet, so lets just be safe and set it to something outside of the range.